msteams-platform/toolkit/create-new-project.md

```diff
@@ -52,7 +52,7 @@ You can create a new app by exploring **View samples** and selecting an existing
   1. Select a sample to browse detailed information.
   1. Select **create** in information page of each sample to download it. 
   1. Run your app locally or remotely to preview in Teams web client by following the instructions which automatically opens after you download the sample.
-  1. If you don’t want to download the samples, you can select **View on GitHub** to open the sample in the GitHub Samples repository and browse the source code.
+  1. If you don’t want to download the samples, you can select **View on GitHub** to open the sample in the GitHub Samples repository and browse the source code. 
```

Comment:
Why add an extra space? It serves no purpose and leads to a markdownlint warning.

Reply:
Corrected. Thanks, Ashish.